README.md

```diff
@@ -91,5 +91,6 @@ Please check [CONTRIBUTING.md](CONTRIBUTING.md) and the [Developer Guide](docs/d
 
 ## Community
 
-* Join our [Kubeflow Slack Channel](https://kubeflow.slack.com/archives/C06627U3XU3)
+* Join [CNCF Slack Channel](https://www.kubeflow.org/docs/about/community/#kubeflow-slack-channels)
+* Check out our blog post [Announcing the Kubeflow Spark Operator: Building a Stronger Spark on Kubernetes Community](https://github.com/kubeflow/blog/blob/master/_posts/2024-04-15-kubeflow-spark-operator.md)
```

Comment:
replace the link with https://blog.kubeflow.org/operators/2024/04/15/kubeflow-spark-operator.html